rhpam-config/7.13.0/common.yaml

```diff
@@ -561,10 +575,24 @@ smartRouter:
                   - name: KIE_SERVER_ROUTER_TLS_KEYSTORE
                     value: "/etc/smartrouter-secret-volume/keystore.jks"
                   #[[end]]
+                  #[[if not .SecretAdminCredentials]]
                   - name: KIE_ADMIN_USER
-                    value: "[[.AdminUser]]"
+                    value: "[[$.AdminUser]]"
```

Comment:
I think it should contain $ here.

Reply:
Should or shouldn't ? Currenty the $ is contained

Reply:
shouldn't, $ means loop, should be only on servers object.

Reply:
fixed